cards/common-partials/thumbsfeedback.hbs

```diff
@@ -0,0 +1,29 @@
+<form class="HitchhikerCard-thumbs js-HitchhikerCard-feedbackForm">
+  <fieldset class="HitchhikerCard-fieldset">
+    <label class="HitchhikerCard-thumb">
+      <span class="HitchhikerCard-thumbUpIcon">
+        {{> icons/builtInIcon iconName='thumb' }}
+      </span>
+      <input type="radio"
```

Comment:
I'm a little surprised these are radio inputs rather than buttons, but since that's the current direct answers feedback works, I think we're good to leave it